ray-operator/controllers/ray/utils/serve_httpclient.go

```diff
@@ -0,0 +1,160 @@
+package utils
+
+import (
+	"bytes"
+	"io/ioutil"
+	"net/http"
+
+	rayv1alpha1 "github.com/ray-project/kuberay/ray-operator/apis/ray/v1alpha1"
+	"k8s.io/apimachinery/pkg/util/json"
+)
+
+var (
+	DeployPath = "/api/serve/deployments/"
+	StatusPath = "/api/serve/deployments/status"
+)
+
+// ServeConfigSpec defines the desired state of RayService, used by Ray Dashboard.
+type ServeConfigSpec struct {
```

Comment:
Not sure if that's exact same as api definition. If so, can we import from apis folder directly?

Reply:
Go code is the same, but the json name is different.

Reply:
...it's because we're trying to use camelCase in the CRD, but the existing REST API accepts only Python-style snake_case.